samples/android/Sample.csproj

```diff
@@ -48,19 +48,13 @@
     <Reference Include="System.Core" />
     <Reference Include="Mono.Android" />
     <Reference Include="Xamarin.GooglePlayServices.Basement">
-      <HintPath>packages\Xamarin.GooglePlayServices.Basement.60.1142.1\lib\MonoAndroid80\Xamarin.GooglePlayServices.Basement.dll</HintPath>
+      <HintPath>packages\Xamarin.GooglePlayServices.Basement.71.1610.0-preview6\lib\monoandroid90\Xamarin.GooglePlayServices.Basement.dll</HintPath>

```

Comment:
The FCM package requires preview versions of GooglePlayServices and Firebase to work. We may want to delay our release until the final versions are released.